base/threadingconstructs.jl

```diff
@@ -186,6 +186,15 @@ For example, the above conditions imply that:
 - The value of [`threadid()`](@ref Threads.threadid) may change even within a single
   iteration.
 
+The keywords `continue` and `break` can be used in conjuction with `@threads`. They have the following meaning:
+
+- `continue` skips the rest of the current iteration.
+- `break` gives julia freedom to skip any iteration not yet executed. 
+  There is however no guarantee about particular iterations being skipped. 
+  Which iterations exectly are skipped is an implementation detail and schedular dependent.
+  Typically the loop is subdivided into blocks and a break statement will only skip the remainder
+  of the surrounding block.
```

Comment:
@Seelengrab you like this formulation better? I still gave julia the freedom to skip whatever it wants, to make the statement more future-proof.